File: MPI/mpicheck/tests/functions/4.0/MPI_Raccumulate_c.f

        
        program main
        include 'mpif.h'
        
        TYPE(INTEGER) var_0(10)
       INTEGER(KIND=MPI_COUNT_KIND) var_1
       INTEGER var_2
       INTEGER var_3
       INTEGER(KIND=MPI_ADDRESS_KIND) var_4
       INTEGER(KIND=MPI_COUNT_KIND) var_5
       INTEGER var_6
       INTEGER var_7
       INTEGER var_8
       INTEGER var_9
       INTEGER var_10
        call mpi_raccumulate_c(var_0, var_1, var_2, var_3, var_4, var_5, var_6, var_7, var_8, &
       var_9, var_10)
       call pmpi_raccumulate_c(var_0, var_1, var_2, var_3, var_4, var_5, var_6, var_7, var_8, &
       var_9, var_10)
        end program main
    

File: MPI/mpicheck/tests/functions/4.0/MPI_Raccumulate_c.c
#include <mpi.h>
int main(char argc, char**argv)
{
    /* vars */
    const void *var_0;
    MPI_Count var_1;
    MPI_Datatype var_2;
    int var_3;
    MPI_Aint var_4;
    MPI_Count var_5;
    MPI_Datatype var_6;
    MPI_Op var_7;
    MPI_Win var_8;
    MPI_Request *var_9;
    int ret;
    /* calls */
    ret = MPI_Raccumulate_c(var_0, var_1, var_2, var_3, var_4, var_5, var_6, var_7, var_8, var_9);
    ret = PMPI_Raccumulate_c(var_0, var_1, var_2, var_3, var_4, var_5, var_6, var_7, var_8, var_9);
    return 0;
}
